src/asm2wasm.h

```diff
@@ -915,6 +978,9 @@ void Asm2WasmBuilder::processAsm(Ref ast) {
         auto* func = pair.second;
         passRunner.runFunction(func);
       }
+      for (auto* func : extraSupportFunctions) {
+        passRunner.runFunction(func);
```

Comment:
the `runFunction` name is confusing. we aren't running the function, we're running the optimizations *on* the function.

Reply:
Good point, thanks. I'll fix in a new PR.